### PR TITLE
blue blood now requires copper instead of iron to cure bloodloss

### PR DESCRIPTION
**Pull request by brainfood1183**
Arachnid species and other blue blooded creatures such as spiders and crabs now require copper to stop bloodloss instead of iron (giving them iron tablets will poison them).

Added - new tag CopperBlood (add to mobs with blue blood to make them require copper for bloodloss).
Renamed - spider blood to hemocyanin blood ( not just for spiders).
Changed - crabs now have blue blood.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- add: Arachnid and other Blue (hemocyanin) blooded creatures now require copper to treat bloodloss instead of iron and brute medkits contain 2 copper pills and 1 less iron pill.

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/Species/arachnid.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     baseDecayRate: 0.0125 # Read comment in hunger
   # Damage (Self)
   - type: Bloodstream
-    bloodReagent: SpiderBlood # would be cool if they had to drink copper instead of iron but that might just be more bloat to deal with.
+    bloodReagent: CopperBlood # would be cool if they had to drink copper instead of iron but that might just be more bloat to deal with.
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Mobs/Species/arachnid.yml:23]: :trollface: it would be cool indeed

Comment (brainfood1183) [reply Resources/Prototypes/Entities/Mobs/Species/arachnid.yml:23]: its in ss13 so it will be so :)

**Review by mirrorcult — CHANGES_REQUESTED** [review]
i'd prefer for a new reagent condition to get added that just checks for blood reagent rather than doing it indirectly through tags and `HasTag`

**Comment by brainfood1183** [comment]
> i'd prefer for a new reagent condition to get added that just checks for blood reagent rather than doing it indirectly through tags and `HasTag`

Might be out of my league but i will give it a go :)

**Comment by brainfood1183** [comment]
> i'd prefer for a new reagent condition to get added that just checks for blood reagent rather than doing it indirectly through tags and `HasTag`

created a new metabolizer type and modified the arachnid heart to metabolize iron and copper differently then applied this new heart to all hemocyanin animals as well.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**